app/Http/Controllers/Admin/AdminController.php

```diff
@@ -15,21 +39,143 @@ class AdminController extends Controller
      */
     public function index()
     {
-        return view('admin.index');
+        $roleAdmin = \App\User::ADMIN;
+        return view('admin.user.dashboard', ['roleAdmin' => $roleAdmin]);
     }
 
+    /**
+     * Log account admin out of the application.
+     *
+     * @return void
+     */
     public function userList()
     {
-        return view('admin.user.list');
+        $users = Auth::user();
+        $roleAdmin = \App\User::ADMIN;
+        if ($users->role === \App\User::ADMIN) {
+            $users = User::where('role', \App\User::ROLE_USER)
+                ->paginate(\App\User::PAGINATE_USER);
+            return view(
+                'admin.user.list', [
+                'users' => $users, 'roleAdmin' => $roleAdmin
+                ]
+            );
+        } else {
+            return redirect()->back();
```

Comment:
This condition is shorten , please reverse the if , avoid  use else

Reply:
Fixed

Reply:
Still have else exist in code